content/en/contribute/_index.md

```diff
@@ -20,4 +20,17 @@ The [Development](https://forum.communityhealthtoolkit.org/c/developement/7) sec
 - [Share your expertise](https://forum.communityhealthtoolkit.org) on our Forum
 - Sign up for our [newsletter](https://communityhealthtoolkit.org/contact) to get the latest updates
 
+**Are you a partner wondering how issues are prioritized?**
+
+Issues that a partner will: 
+
+* Upgrade to get the feature/bugfix
+* Help validate, design and prototype the feature 
+* Help fund the development
+* Be blocked by it for a major release or expansion 
+* Has users that are unable to use the CHT or experience it in a way that results in providing bad care
+
+Will get prioritized.  Futher, internally, a ticket may be needed to facilitate a release or reduce technical debt.
```

Comment:
A spelling correction and suggestion for wording of tech debt.

```suggestion
Will get prioritized.  Further, internally, a ticket may be needed to facilitate a release or perform important upgrades as is important to do in the maintenance of software.
```

Reply:
The close proximity of the two "important"s in your suggestion feels awkward. How about:

```suggestion
Will get prioritized.  Further, internally, a ticket may be needed to facilitate a release or perform important upgrades as is critical to do in the maintenance of software.
```